src/enums.ts

```diff
@@ -0,0 +1,3 @@
+enum RestrictionLevel {
+    HIGHLY_RESTRICTIVE,

```

Comment:
Not sure if you're planning on implementing the other restriction levels, but if not, i'd suggest removing this and making the script `HIGHLY_RESTRICTIVE` as the only option